src/Apps/Sell/Routes/IntroRoute.tsx

```diff
@@ -1,73 +1,108 @@
+            {introData.map((intro, index) => (
+              <Flex key={index} justifyContent="space-between">
+                <Text variant={["sm-display", "lg"]}>{index + 1}</Text>
+                <Box px={1}>
+                  <Text variant={["sm-display", "lg"]}>{intro.title}</Text>
+                  <Text variant={["xs", "sm"]} color="black60">
+                    {intro.description}
+                  </Text>
+                </Box>
+                <Image
+                  src={intro.image}
+                  srcSet={intro.image}
+                  alt={`Step ${index + 1}`}
+                  width={[70, 100]}
+                  height={[70, 100]}
+                  style={{ objectFit: "cover" }}
+                />
+              </Flex>
+            ))}
+          </Join>
 
-  return (
-    <AppContainer>
-      <SubmissionHeader />
-      <Flex py={4} flexDirection="column" alignItems="center">
-        <Box width={600}>
-          <Text variant="lg-display">It’s easy to sell on Artsy</Text>
-          <Spacer y={2} />
-          <Flex justifyItems="space-between">
-            <Box>
-              <Text variant="md">Tell us about your work</Text>
-              <Text variant="sm" color="black60">
-                Start by adding an artist from our list of high demand artists.
-                Include information such as year, medium, dimensions and
-                materials.
-              </Text>
-            </Box>
-            <SkeletonBox width={100} height={100} />
-          </Flex>
-          <Spacer y={2} />
-          <Flex justifyContent="space-between">
-            <Box>
-              <Text variant="md">Upload arwork images</Text>
-              <Text color="black60">
-                Improve your chances of selling by including photographs of the
-                front, back, frame, signature and other details.
-              </Text>
-            </Box>
-            <SkeletonBox width={100} height={100} />
-          </Flex>
-          <Spacer y={2} />
-          <Flex justifyContent="space-between">
-            <Box>
-              <Text variant="md">Complete submission</Text>
-              <Text color="black60">
-                Your work will be submitted to an Artsy advisor who will assess
-                whether your work is eligible and help guide you on next steps.
-              </Text>
-            </Box>
-            <SkeletonBox width={100} height={100} />
-          </Flex>
-          <Spacer y={2} />
-          <Flex justifyContent="space-between" gap={2}>
-            <Button onClick={onCreateSubmission}>New Submission</Button>
+          <Spacer y={6} />
 
+          <Flex
+            gap="2"
+            width="100%"
+            bg="white100"
+            position={isMobile ? "fixed" : "relative"}
+            bottom={isMobile ? 0 : undefined}
+            left={isMobile ? 0 : undefined}
+            px={isMobile ? 2 : 0}
+            py={isMobile ? 4 : 0}
```

Comment:
Fixed button on mWeb at the bottom. This is for now but we should have a separate component that wraps this button and the footer navigation.